dissect/volume/ddf/ddf.py

```diff
@@ -7,37 +7,44 @@
 
 from dissect.volume.ddf.c_ddf import DEFAULT_SECTOR_SIZE, c_ddf
 from dissect.volume.exceptions import DDFError
-from dissect.volume.raid.raid import RAID, Configuration, PhysicalDisk, VirtualDisk
+from dissect.volume.raid.raid import (
+    RAID,
+    Configuration,
+    DiskMap,
+    PhysicalDisk,
+    VirtualDisk,
+)
 from dissect.volume.raid.stream import Layout, Level
 
+DDFPhysicalDiskDescriptor = Union[BinaryIO, "DDFPhysicalDisk"]
```

Comment:
```suggestion
DDFPhysicalDiskDescriptor = BinaryIO | DDFPhysicalDisk
```

Reply:
The `|` operator doesn't work with strings unfortunately.

Reply:
What if you put this in a `if TYPE_CHECKING`?

Reply:
That sucks that it still doesn't allow you to do it unquoted, figured the typing system was mature enough for that by now :frowning_face: